Extension/browser/safari/lib/converter.js

```diff
@@ -307,7 +307,8 @@ var SafariContentBlockerConverter = {
 
             function isUrlBlockRule(r) {
                 return self._isContentType(r, adguard.rules.UrlFilterRule.contentTypes.URLBLOCK) ||
-                    self._isContentType(r, adguard.rules.UrlFilterRule.contentTypes.GENERICBLOCK);
+                    self._isContentType(r, adguard.rules.UrlFilterRule.contentTypes.GENERICBLOCK) ||
```

Comment:
I don't want to get rid of `if-document`.

We should first find out if it is possible to use `url-filter`